python/ray/dag/compiled_dag_node.py

```diff
@@ -586,9 +641,28 @@ def __init__(self):
                 super().__init__(daemon=True)
                 self.in_teardown = False
 
+            def wait_teardown(self):
```

Comment:
Is this a part of other PR?

Reply:
Ah yes, probably needs to get fixed in merge.